pkg/validation/upstreams.go

```diff
@@ -70,7 +70,7 @@ func validateStaticUpstream(upstream options.Upstream) []string {
 	if upstream.InsecureSkipTLSVerify {
 		msgs = append(msgs, fmt.Sprintf("upstream %q has insecureSkipTLSVerify, but is a static upstream, this will have no effect.", upstream.ID))
 	}
-	if upstream.FlushInterval != nil && *upstream.FlushInterval != time.Second {
+	if upstream.FlushInterval != nil && upstream.FlushInterval.Duration() != time.Second {
```

Comment:
I know you are just refactoring a bit of code, but this caught my eye:

It's a little unclear in the code that `time.Second` is a special value here because it aligns with the default `flushInterval` value of `1s`.

Maybe a comment is all this needs to make it more clear?

Reply:
I've added a constant to the options package, seems the only sensible place to put the constant. Then I've used that wherever we were using the magic 1 second value, WDYT? (That's all contained in the final commit)